src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -247,6 +247,74 @@ void HexagonGame::initLua_AudioControl()
         .doc(
             "Dives into the `Sounds` folder of the current level pack and "
             "plays the specified file `$0`.");
+
+    addLuaFn("a_syncMusicToDM", //
+        [this](bool value) {
+            levelStatus.syncMusicToDM = value;
+
+            auto* current(assets.getMusicPlayer().getCurrent());
+            setMusicPitch(current);
```

Comment:
Shouldn't this change `levelStatus.syncMusicToDM` instead of calling `setMusicPitch`?